gateway-messages/src/sp_to_mgs.rs

```diff
@@ -570,6 +835,9 @@ pub enum SpError {
     Update(UpdateError),
     Sensor(SensorError),
     Vpd(VpdError),
+    BlockOutOfOrder,
+    InvalidSlotIdForOperation,
+    InvalidComponent,
```

Comment:
Why are these variants added both here and to `UpdateError`?

Reply:
Bump to include this question in this review

Reply:
Neglected to delete from SpError during development apparently. Thanks.